Oqtane.Server/appsettings.json

```diff
@@ -1,5 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Data Source=(LocalDb)\\MSSQLLocalDB;AttachDbFilename=|DataDirectory|\\Oqtane.mdf;Initial Catalog=Oqtane;Integrated Security=SSPI;"
+    "DefaultConnection": "Data Source=(LocalDb)\\MSSQLLocalDB;AttachDbFilename=|DataDirectory|\\Oqtane-201905122226.mdf;Initial Catalog=Oqtane-201905122226;Integrated Security=SSPI;"
```

Comment:
This file should not be included in the pull request. When the application is run for the first time, it modifies the appsettings.json to include a unique name for the LocalDB database so that it does not conflict with any existing database on a users machine.

Reply:
Fixed that one